eo2js/test/commands/transpile.test.js

```diff
@@ -68,6 +87,25 @@ describe('transpile', function() {
         assertFilesExist(transpile(name), target, [`project/com/eo2js/${name}.test.js`])
       })
     })
+    it('should skip transpilation if source was not modified', function() {
+      transpile()
+      const transpiled = path.resolve(target, '8-transpile/com/eo2js/simple.xmir')
+      const first = fs.statSync(transpiled).mtime
+      retranspile()
+      const second = fs.statSync(transpiled).mtime
+      assert.equal(first.getTime(), second.getTime())
+    })
+    it('should retranspile if source was modified', async function() {
+      transpile()
+      const transpiled = path.resolve(target, '8-transpile/com/eo2js/simple.xmir')
+      const source = path.resolve(target, '6-verify/com/eo2js/simple.xmir')
+      const first = fs.statSync(transpiled).mtime
+      await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
@HaidarJbeily7 why do we need timeout here?

Reply:
@maxonfjvipon  Testing on windows might fail without deterministic waiting time here as for a case when a file is updated in a small timespan, it may result test failing here. (first and second mtime are equal)

https://blog.andreiavram.ro/time-precision-linux-windows/